packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_indirect/TestLibraryIndirect.py

```diff
@@ -12,7 +12,14 @@
 """
 Test `@_implementationOnly import` behind some indirection in a library used by the main executable
 """
-import commands
+
+#SWIFT_TENSORFLOW: Since tensorflow uses Python 3 and commands is deprected
```

Comment:
Minor: our "`tensorflow`-specific change" marker is called `SWIFT_ENABLE_TENSORFLOW`.
```suggestion
# SWIFT_ENABLE_TENSORFLOW: Since tensorflow uses Python 3 and commands is deprecated
```

This won't be relevant if you submit the patch upstream. 🙂